src/components/common/Certify.vue

```diff
@@ -125,6 +124,10 @@ export default class Certify extends Vue {
   font-size: $px-14;
 }
 
+.date {
+  color: $gray7;
+}
```

Comment:
Adding this class is fine (since you can't reuse "certify-clause" because it's used for other elements), but I recommend a more description (and longer) name than "date".